kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/RenderFormServiceIntegrationTest.java

```diff
@@ -430,12 +447,311 @@ public void testRenderCaseFormViaUIClientTestBootstrap() throws Exception {
         assertThat(result).contains("<button type=\"button\" class=\"btn btn-success\" onclick=\"startCase(this);\">Submit</button>");
     }
     
+    @Test
+    public void testRenderProcessFormViaUIClientTestMultiSubForm() throws Exception {
+        String result = uiServicesClient.renderProcessForm(CONTAINER_ID, PLACE_ORDER_PROCESS_ID);
+        logger.debug("Form content is '{}'", result);
+        assertThat(result).isNotNull().isNotEmpty();
+        
+        // it has the patternfly (default renderer) css
+        assertThat(result).contains("/files/patternfly/css/patternfly.min.css\" rel=\"stylesheet\">");
+        assertThat(result).contains("/files/patternfly/css/patternfly-additions.min.css\" rel=\"stylesheet\">");
+        
+        // it has required js files
+        assertThat(result).contains("/files/patternfly/js/jquery.min.js\"></script>");
```

Comment:
A suggestion - with AssertJ you can chain the assertions over a single object (in this case `result`) in a builder-like style without repeating `assertThat` on every line.